src/components/Charts/ChartVaccinations.tsx

```diff
@@ -68,7 +74,7 @@ const VaccinesTooltip: React.FC<{
   const pointCompleted =
     seriesCompleted && findPointByDate(seriesCompleted.data, date);
   const pointInitiated = findPointByDate(seriesInitiated.data, date);
-  const isCapped = pointInitiated?.y >= 0.95 ?? false;
+  const isCapped = pointInitiated?.y >= VACCINATION_PERCENTAGE_CAP ?? false;
```

Comment:
nit: I just realized the `?? false` is probably nonsensical / unreachable.  If `pointInitiated` is null/undefined then `pointInitiated?.y` will be undefined, but `undefined >= 0.95` will be `false` and so the `?? false` will never kick in... unless I'm missing something.  So I'd remove it to avoid confusion.

To make the code slightly cleaner and not rely on the behavior of `>=` on `undefined`, you could rewrite it as `(pointInitiated?.y ?? 0) >= VACCINATION_PERCENTAGE_CAP`

Reply:
ah yeah that makes more sense.

I think I was misinterpreting the original logic as "if `pointInitiated.y` is undefined then skip the comparison and return `false`". 

made this change 🙏